pkg/otlp/metrics/metrics_remapping_test.go

```diff
@@ -155,11 +190,23 @@ func TestRemapMetrics(t *testing.T) {
 				metric("system.swap.used",
 					point{f: 2, attrs: map[string]any{"state": "used"}},
 				),
+				metric("system.paging.usage",
+					point{f: divMebibytes * 1, attrs: map[string]any{"state": "free"}},
+					point{f: divMebibytes * 2, attrs: map[string]any{"state": "used"}},
+					point{f: 3, attrs: map[string]any{"state": "buffered"}},
+				),
+			},
+		},
+		{
+			in: metric("system.filesystem.utilization", point{f: 15}),
+			out: []pmetric.Metric{
+				metric("system.disk.in_use", point{f: 15}),
+				metric("system.filesystem.utilization", point{f: 15}),
 			},
 		},
 		{
-			in:  metric("system.filesystem.utilization", point{f: 15}),
-			out: []pmetric.Metric{metric("system.disk.in_use", point{f: 15})},
```

Comment:
Hmmm I feel I'm getting more confused now - looks like the previous test is doing the the right thing, but how comes the resulting metric in datadog exporter got an `otel.` prefix? https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23445#discussion_r1233776214